py_gtfs_rt_ingestion/lib/convert_gtfs_rt.py

```diff
@@ -83,11 +85,14 @@ def process_files(self) -> Iterable[pyarrow.table]:
         iterate through all of the files to be converted, yielding a new table
         everytime the timestamps cross over an hour.
         """
-        table = self.detail.empty_table()
+        table = pyarrow.table(
+            self.detail.empty_table(),
+            schema=self.detail.export_schema,
+        )
```

Comment:
go back to old table coversion method, using all `dict` objects may have been causing run away memory use